src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,78 @@
+package duke;
+
+import duke.command.Command;
```

Comment:
Overall code quality is really good and coding standards are followed (other than the indentation fault in the case-switch section that was previously pointed out.